moon/powerups/server/regen.moon

```diff
@@ -52,6 +52,9 @@ class RegenPowerup extends BasePowerup
         -- Make sure they don't have more 100 HP
         plyHealth = @owner\Health!
         if plyHealth > 100
-            @owner\SetHealthj 100
+            @owner\SetHealth 100
+
+        -- TODO: Should the PowerupManager do this?
+        @owner.Powerups[POWERUP_ID] = nil
```

Comment:
maybe this should be in the BasePowerup `Remove` method

Reply:
We also need to retain the reference to `POWERUP_ID` (local to this file) somehow